### PR TITLE
docs: update readme with a few more first time setup details

### PR DESCRIPTION
**Pull request by tublitzed**
* Mainly opening this to update the mysql connection string url with relevant details for username/password. Made a few other minor tweaks while I was in here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,21 +15,24 @@ and can be run against a local server, e.g.:
...
  * MySQL 5.7 (or compatible)
-   * libmysqlclient (brew install mysql on macOS, apt-get install libmysqlclient-dev on Ubuntu)
+   * libmysqlclient (`brew install mysql` on macOS, `apt-get install libmysqlclient-dev` on Ubuntu)
 
 ## Setting Up
 
-1) [Install Rust]
-
-2) Create a `syncstorage` user/database
+1) [Install Rust](https://www.rust-lang.org/tools/install)
```
Comment (jrconlin) [inline README.md]: Could also use `https://rustup.rs.`

Comment (tublitzed) [reply README.md]: ah, sure. I can change it if that's the preferred way to install?

Comment (jrconlin) [reply README.md]: Sadly, depends on who you ask. Go with your original URL for now. If we hear enough folk say otherwise, we can change it.

**Comment by jrconlin** [comment]
Looks like this commit isn't signed yet. Waiting for that.

File: README.md (hunk 1 above)
Comment (fzzzy) [reply README.md]: Personally I'd say use rustup.

**Comment by tublitzed** [comment]
Gonna just close and re-open this to avoid the mess of un-signed commits. Re-opening with GPG keys setup for all commits.

**Pull request closed without merging**